device-types/QNAP/TS-432PXU-RP.yaml

```diff
@@ -0,0 +1,42 @@
+---
+manufacturer: QNAP
+model: TS-432PXU-RP
+slug: qnap-ts-432pxu-rp
+part_number: TS-432PXU-RP
+u_height: 1
+airflow: front-to-rear
+front_image: true
+rear_image: true
+is_full_depth: true
+subdevice_role: parent
+device-bays:
+  - name: Disk 1
+  - name: Disk 2
+  - name: Disk 3
+  - name: Disk 4
+module-bays:
+  - name: PCIe1
+    position: PCIe1
+comments: '[TS-432PXU-RP | Hardware Specs | QNAP](https://www.qnap.com/en/product/ts-432pxu-rp/specs/hardware)'
+power-ports:
+  - name: PSU1
+    type: iec-60320-c14
+    maximum_draw: 250
+    allocated_draw: 54
+  - name: PSU2
+    type: iec-60320-c14
+    maximum_draw: 250
+    allocated_draw: 54
```

Comment:
As this has removable PSUs they should be modelled under `module-bays` and then added the required PSU modules if they don't already exists in the `module-bays/QNAP/`

Reply:
Addressed in https://github.com/netbox-community/devicetype-library/pull/2521/commits/ec98feeb938fde0a23f9abe26b2d8437cb09d275

So far as I can tell, QNAP use off-the-shelf Delta PSUs.  I can't find any sign of a QNAP-specific SKU or P/N.